services/friendbot/internal/friendbot_handler.go

```diff
@@ -57,7 +57,9 @@ func (handler *FriendbotHandler) doHandle(r *http.Request) (*horizon.Transaction
 		}
 	}
 
-	return handler.loadResult(address, amount)
+	isFundRequest := r.URL.Path == "/fund"
```

Comment:
will this work with trailing backslash as well i.e. `/fund/`? maybe the path should be normalized before checking

Reply:
Other than comparing against `/fund` and `/fund/`, how would I do this in golang?

Reply:
@avi-kik I'm sure you can google a solution for this. here's what I found: https://golang.org/pkg/strings/#HasPrefix

Reply:
@avi-kik I'm going to merge this, but it's on you to fix this.